ur_rtde_msgs/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>ur_rtde_msgs</name>
```

Comment:
Theoretically I think we'd want these messages in `ur_msgs` ([it has some similar messages](https://github.com/ros-industrial/ur_msgs/blob/melodic-devel/msg/RobotStateRTMsg.msg)). Given that that's yet another repo which we don't even currently track as a submodule I don't think it would make sense to actually fork it but can we call this package `ur_extra_msgs` (or `supplement`, `plus`, `more`) and reference that these are our own additions to `ur_msgs`?

Reply:
Renamed `ur_rtde_msgs` -> `ur_extra_msgs`.